dashboard/db/schema.rb

```diff
@@ -531,7 +531,7 @@
     t.string   "instructions"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "phase"
+    t.text     "phase"
```

Comment:
This should go away. This was just an experimental run to see whether we like it what it looks like, before sending it out for actual review I'll sync and rake migrate etc.